WNPRC_EHR/resources/queries/study/drug.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2010-2014 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+require("ehr/triggers").initScript(this);
+var WNPRC = require("wnprc_ehr/WNPRC").WNPRC;
+
+function onInit(event,helper){
+    helper.decodeExtraContextProperty('clientEncounterDate');
+
+    helper.registerRowProcessor(function(helper,row) {
+        if (!row)
```

Comment:
@dnicolalde The code here I've seen in more than 2 trigger scripts, could probably be ripped out and placed into a reusable "utils" function but just going to note it here for now.